### PR TITLE
docs - notes on block permission

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
- Adds a note to the blocks permissions section to clarify what data permissions are required in order to overrule a block.
- Adds notes about what exactly is meant by "Unrestricted data access".

Will likely follow up with a short Learn article that walks through an example of block in action.

See also: https://github.com/metabase/metabase/issues/21695

### CODE REVIEW

File: docs/administration-guide/data-permissions.md [hunk 1]
```diff
@@ -24,54 +24,61 @@ You can set various levels of permissions on a data source, from querying access
 
 ## Data access
 
-You can click on any cell in the permissions table to change a group’s access level. When you’re done making your changes, just click the **Save changes** button in the top-right, and you’ll see a confirmation dialog summarizing the changes.
+You can click on any cell in the permissions table to change a group’s access level. When you’re done making your changes, just click the **Save changes** button in the top-right, and you’ll see a confirmation dialog summarizing the changes.
```
Comment (nemanjaglumac) [inline docs/administration-guide/data-permissions.md:27]:
```suggestion
You can click on any cell in the permissions table to change a group’s access level. When you’re done making your changes, just click the **Save changes** button in the top-right, and you’ll see a confirmation dialog summarizing the changes.
```

**Review by brunobergher — COMMENTED** [review]
Minor comment

File: docs/administration-guide/data-permissions.md [hunk 2]
```diff
@@ -24,54 +24,61 @@ You can set various levels of permissions on a data source, from querying access
...
 
 ### Granular access
 
-**Granular access** allows administrators to explicitly set access to tables or schemas within a database. In practice, this means that:
+**Granular access** allows administrators to explicitly set data access to tables or schemas within a database, with "data access" here meaning the ability to create questions using the graphical query builder. In practice, this means that:
 
-- Admins can set the groups access to individual tables to either **Unrestricted**, **No self-service**, or **Sandboxed** access.
+- Admins can set the group's access to individual tables to either **Unrestricted**, **No self-service**, or **Sandboxed** access.
 - If a new table gets added to this database in the future, the group won't get access to that new table. An administrator would need to explicitly grant access to that table.
```
[the remark this answers is not in the document]
Comment (nemanjaglumac) [reply docs/administration-guide/data-permissions.md:38]: This makes a lot of sense and it doesn't hurt to mention it one more time.

**Review by brunobergher — COMMENTED** [review]
Looks good to me, but I'd love for someone with more IQ and attention to detail to look at it, like @nemanjaglumac .

**Review by nemanjaglumac — APPROVED** [review]
👌 

So much clarity!
Thank you

File: docs/administration-guide/data-permissions.md [hunk 3]
```diff
@@ -28,50 +28,59 @@ You can click on any cell in the permissions table to change a group’s access
...
 
 ### Block access
 
 {% include plans-blockquote.html feature="Block access" %}
 
-**Block** ensures people can’t ever see the data from this database, regardless of their permissions at the Collection level. So if they want to see a question in a collection that have access to, but that question uses data from a database that's been blocked for that person's group, then they won't be able to see that question.
+**Block** ensures people in a group can’t see the data from this database, regardless of their permissions at the collection level. 
+
+Even if a question is in a collection that the group has access to, but that question queries a database that is blocked for that group, people in that group won't be able to view that question _unless_ they're in another group with the relevant data permissions. Essentially, what Block does is make collections permissions insufficient to view a question.
```
Comment (nemanjaglumac) [inline docs/administration-guide/data-permissions.md:52]: > Essentially, what Block does is make collections permissions insufficient to view a question.

@jeff-bruemmer do you think some people might be confused by this and expect that they will either 1) not see a question in a collection or 2) not be able to open it?

The crucial part here is that they are not able to see the data in that question.
They can still see question's name and description (and they can even change those if they have Curate permission level on a collection).

**Comment by jeff-bruemmer** [comment]
@metabase-bot backport release-x.43.x

**Pull request merged**